src/20_Components/amp-date-countdown.html

```diff
@@ -32,7 +32,7 @@
     The number of days, hours, minutes and seconds remaining is bound to `{{d}}`, `{{h}}`, `{{m}}` and `{{s}}` respectively.
     Double digit format variants (`{{dd}}`, `{{hh}}`, `{{mm}}` and `{{ss}}`) are also available.
   -->
-  <amp-date-countdown timestamp-seconds="2147483648" layout="fixed-height" height="50">
+  <amp-date-countdown timestamp-seconds="2147483648" layout="flex-item">
```

Comment:
I find using the `flex-item` layout here a bit "magic". Can you either explain why you're using it or use another layout?